hubgrep/config/__init__.py

```diff
@@ -45,6 +45,8 @@ class Config:
 
     WATCH_SCSS = False
 
+    HOSTER_REQUESTS_TIMEOUT = int(os.environ.get('HUBGREP_HOSTER_REQUESTS_TIMEOUT', 2))
```

Comment:
should probably get in the habit of always calling them HOSTER_SERVICE_xxx to be consistent

Reply:
done!